requirements.in

```diff
@@ -17,3 +17,6 @@ GitPython==3.1.2
 cattrs==1.0.0
 attrs==19.3.0
 typing==3.7.4.1
+click==7.1.2
+zipp==3.1.0
+importlib-metadata==1.7.0
```

Comment:
I don't think`zipp` and `importlib-metadata` should be added here. They are transitive dependencies with python <= 3.7, but we don't directly reference them anywhere in this repo.